OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/UploadKeyActivity.java

```diff
@@ -89,15 +90,6 @@ public void onClick(View v) {
             return;
         }
 
-        try {
```

Comment:
Because now there is no need of getting masterKeyId from DataUri. As I am sending masterKeyId in Intent itself while calling this activity itself.  So, this try-cache is moved there.  Please see ViewKeyAdvShareFragment.java where I am calculating masterKeyId from dataUri and sending to this activity with Intent.